.gitignore

```diff
@@ -57,6 +57,12 @@
 /test/Compiler/with-stdlib/AllStdLib
 /test/Compiler/with-stdlib/agda-rts.js
 /test/Compiler/with-stdlib/highlight-hover.js
+/test/interaction/*.out
```

Comment:
Not these.  These are our golden values. We track the output produced by Agda in these files.

Reply:
Checked the files in instead